src/go/gcsrunner/transform_config.go

```diff
@@ -101,29 +98,59 @@ func transformConfigBytes(config []byte, opts FetchConfigOptions) ([]byte, error
 
 func transformEnvoyConfig(bootstrap *bootstrappb.Bootstrap, opts FetchConfigOptions) error {
 	listeners := bootstrap.GetStaticResources().GetListeners()
-	if len(listeners) != 1 {
-		return fmt.Errorf("expected exactly 1 listener, got: %d", len(listeners))
+	if len(listeners) == 0 {
+		return fmt.Errorf("expected at least 1 listener, got: 0")
+	}
+	// Require http_listener or https_listener, as the absence of these incidates
+	// a broken config.
+	// Note that loopback_listener is not required.
+	httpListenerTransformed := false
+	for _, l := range listeners {
+		switch l.GetName() {
+		// The default listener created by this project's configgenerator package.
+		case "http_listener", "https_listener":
```

Comment:
should we call it "ingress_listener", vs loopback_listner?  really don't care about http, https or grpc

Reply:
The reason for this is because it's hard-coded into configgenerator. `ingress_listener` does make sense, but whatever name we decide, we might want to make this a constant shared between configgenerator and this package.

Reply:
If we don't have backward compatible issue, I like to make such change.